configure

```diff
@@ -8632,7 +8632,7 @@ cat >>$CONFIG_STATUS <<_ACEOF || ac_write_fail=1
     # Replace #undef with comments.  This is necessary, for example,
     # in the case of _POSIX_SOURCE, which is predefined and required
     # on some systems where configure will not decide to define it.
-    if (defundef == "undef") {
+    if (defundef == "undef" && macro !~ /^OMR/) {
```

Comment:
This is a derived artifact: this change will be overwritten if `HAS_AUTOCONF=1` is in the build environment.